Dockerfile

```diff
@@ -94,6 +98,14 @@ EXPOSE 30000/TCP
 # EXPOSE 33478/UDP
 # EXPOSE 49152-65535/UDP
 
+COPY --from=optional-release-stage /root/status.txt .
+
+RUN \
+  if  [[ $(cat status.txt) = "installed" ]]; then \
+    source ./apply_patches.sh; \
```

Comment:
We may want to remove the `./` from here too, just to be consistent.

Reply:
Please also rebase your change as it's outdated with target branch as per `This branch is out-of-date with the base branch`

Reply:
Will do sometime soon.